projects/storefrontapp/server.ts

```diff
@@ -63,7 +63,7 @@ export function app() {
 }
 
 function run() {
-  const port = process.env.PORT || 4000;
+  const port = process.env['PORT'] || 4000;
```

Comment:
Since angular13, new generated shellapp through ng cli provides by default the `noPropertyAccessFromIndexSignature` property, which won't allow access to index signatures using `dots`, but only through `indexed`